### ISSUE TITLE
Lakea Receive Events from Woodland

### ISSUE THREAD

**Issue by JohnWolfson** [JohnWolfson/Lakea-Stream-Assistant#11]
Lakea needs to be able to receive event triggers from the Woodland

Acceptance Criteria:

- Lakea can receive a message from the Woodland and log it to console

**Comment by JohnWolfson** [comment]
This ticket is currently blocked, waiting on the results of https://github.com/JohnWolfson/Lakea-Stream-Assistant/issues/7 before implementing. It should be noted that the woodland is currently a prototype and a new project for this will be created at some point, so this ticket may not be tackled until then.

**Comment by IstalriSkolir** [comment]
The Stream Assistant now has a websocket that the new Woodland project will be able to connect to when its started. Further service endpoints still need implementing, however these will be reflected in new tickets. Archiving this ticket as its been superseded by these changes.
